unikernel.ml

```diff
@@ -751,7 +840,37 @@ struct
                 ~icon:"/images/robur.png" ())
              `Unauthorized)
 
-  let users store reqd (user : User_model.user) =
+  let close_session store reqd ~json_dict (user : User_model.user) =
```

Comment:
I struggle with `close_sessions` vs `close_session` -- what is the idea behind having two separate functions?

Reply:
so `close_sessions` just uses the users session cookie and deletes every other cookie except the one making the request while `close_sesssion` has a json body in the request with the specific cookie the user wants to delete.

Reply:
I still don't understand why there used to be one function and now we have multiple functions.

Reply:
so previously one of the functions `close_session` was a `GET` request and when doing this token authentication it was unclear to me at the time which of the requests will go via the API, so I changed this into a `POST` request and now the cookie is sent in the json body.  Given that the endpoints are more clearly defined now I think this change can be reverted

Given that